src/main/java/io/reactiverse/pgclient/PgConnectOptions.java

```diff
@@ -233,7 +248,12 @@ public PgConnectOptions setIdleTimeout(int idleTimeout) {
 
   @Override
   public PgConnectOptions setSsl(boolean ssl) {
-    return (PgConnectOptions)super.setSsl(ssl);
+    if (ssl) {
+      setSslMode(SslMode.REQUIRE);
```

Comment:
does REQUIRE validates the certificates ?

Reply:
I believe it is rather equivalent to verify_ca

Reply:
It's stated in https://www.postgresql.org/docs/current/libpq-connect.html

> only try an SSL connection. If a root CA file is present, verify the certificate in the same way as if verify-ca was specified

Reply:
Why setSsl(true) is equivalent to verify_ca?